utils/build-presets.ini

```diff
@@ -963,6 +963,30 @@ mixin-preset=
     buildbot_linux,no_assertions
     mixin_buildbot_linux,no_test
 
+[preset: mixin_lld]
+# Configure the built toolchain to use lld as the default linker
+extra-cmake-args=-DCLANG_DEFAULT_LINKER=lld
```

Comment:
I think the spelling is `extra-cmake-options`, but maybe use `swift-cmake-options` to match the LLVM ones below?

Reply:
I'd looked for a `swift-cmake-options`, but didn't see one. You're right though, it's `extra-cmake-options`.

Reply:
There's one in https://github.com/apple/swift/blob/main/utils/build-presets.ini#L1276